server/router/main.js

```diff
@@ -0,0 +1,36 @@
+module.exports = function(app) {
```

Comment:
I think this file should be located in server/push.
Or, we can consider 'public' directory to provide rest APIs to upper layer(e.g. client layer).